templates/security/cve/index.html

```diff
@@ -68,21 +68,22 @@ <h3 class="p-heading--5">By Ubuntu release</h3>
         </div>
         <div class="col-7">
           <ul class="p-inline-list" style="margin-top: .3rem;">
-            <li class="p-inline-list__item"><a>All maintained releases</a></li>
-            <li class="p-inline-list__item"><a>All LTS</a></li>
-            <li class="p-inline-list__item"><a>22.04 LTS</a></li>
-            <li class="p-inline-list__item"><a>20.04 LTS</a></li>
-            <li class="p-inline-list__item"><a>18.04 LTS</a></li>
-            <li class="p-inline-list__item"><a>23.10</a></li>
-            <li class="p-inline-list__item"><a>Other releases</a></li>
+            <!-- Once the search results page is finished and the cve list is not tied to the current table cols, selected releases, maintained releases, and lts releases will be accounted for separately -->
```

Comment:
Sorry, I couldn't understand this comment. Can you please add more details here?

Reply:
This and the above comment are related, currently the cve table loops through `selected_releases` to decide which releases are going to be column headers, but in the new design for these links we really need `maintained_releases` (which is basically the same as `selected_releases`) and `lts_releases`. I didn't think it was worth it to spend time changing the way the current table works just to test that these links are correct, given that that table isn't going to be used anymore, so I added the comment explaining why lts release links are hard coded but maintained are not